### ISSUE TITLE
Not clear how to actually create or update an existing issue?

### ISSUE THREAD

**Issue by rmflight** [MilesMcBain/issuecreep#22]
This is very, very cool.

However, what I can't figure out is the actual command to actually create a new issue.

If I do:

- rmdgh::draft_issue("user/repo")
- That opens a new file for me to fill out the specifics, I save it.
- What command is next? rmdgh::github_issue(..., action = "create")?? But it doesn't take a filename argument ...

There really needs to be one more step noted in the README, what do you do **after** you've updated the Rmd file.

**Comment by MilesMcBain** [comment]
In the readme I had this:

> We have an Rmarkdown output format called github_issue that can be used to submit issues, issue updates, and issue comments to GitHub on document render

Where 'on document render' refers to `rmarkdown::render()` or the 'knit' button in RStudio or knit command in VSCode.

**Comment by MilesMcBain** [comment]
I agree this could be more clear!

**Comment by MilesMcBain** [comment]
I addes some stuff in [42aa56f](https://github.com/MilesMcBain/rmdgh/commit/42aa56f3b6ded9bd6e32928464f9dbbb32d3bfe2) to the readme and documentation for `draft_issue()`. Does that help clarify do you think or does it need more treatment in the README?

**Comment by rmflight** [comment]
Yes, that helps, I think. I didn't actually double check my repo after changing to `draft = false` and knit'ing yesterday, and noticed that the issue was there.

What about adding a verbose option to rmdgh, that will remind the user that they rendered a draft issue, and it's not submitted, and also gave feedback that the issue was created, commented, etc.

I think that was part of my confusion, I didn't have anything to remind me that it was just showing draft vs actually creating / updating to the actual issue.

I could work on a PR implementing that next week possibly.

**Comment by MilesMcBain** [comment]
There is already is a message on draft: yes, but it looks much less
prominent in RStudio than VSCode where I was developing.

Part of the issue is that RStudio pops open the rendered .md file over the
rendering output where you would encounter the message.

I'm happy to look at ideas to fix this.

**Comment by MilesMcBain** [comment]
Actually that's not fully correct, an .md doesn't open on draft: yes, only draft: no. 

There is a message in both cases. On draft: no it tells you an issue was submitted and displays the issue URL.

**Comment by rmflight** [comment]
OK, I'll try it again today or tomorrow and see what happens again. Maybe I just wasn't paying good enough attention. Thanks so much for humoring me!

**Comment by MilesMcBain** [comment]
I think something that might work is to also inject a message and link into the rendered MD output that pops up over the top of the message and link and the render pane.

**Comment by rmflight** [comment]
I think adding something at the beginning or end of the rendered MD would be good, or alternatively, use an actual `message()` in the console, or fancier using `cli` functions. The advantage of putting it there is it shouldn't matter which IDE is used.

When the rendered MD shows up in my render pane of RStudio, I don't have a chance to see the message at all. This is on v 2022.11.0 Build 100 of RStudio.

**Comment by rmflight** [comment]
Aha, I finally see it. It's in the "render" pane. Wow. Yeah, that gets hidden too easily in RStudio as it jumps from doing the rendering to displaying the output.

After all that, I wonder if it's necessary to add anything else. Maybe a note in the README to **check** the render pane for any messages if something is wrong.

**Comment by MilesMcBain** [comment]
Okay cool we'll try something in the MD output.

I am using message() but all output seems to be directed to the render
pane. I think this is because a separate session is used to render. So I
don't think the console is possible using the 'knit to submit' approach.
